pkg/kubectl/cmd/exec.go

```diff
@@ -48,6 +48,10 @@ var (
 		kubectl exec 123456-7890 -c ruby-container -i -t -- bash -il`)
 )
 
+const (
+	execUsageStr = "expected 'exec <pod-name> <command> [arg1] [arg2] ... [argN]'.\n<pod-name> and <command> are required parameter for the exec command"
```

Comment:
We generally use UPPERCASE argument names pretty consistently across all the commands.  Random sample of kubectl shows we use it for values to define the type everywhere in usage.  I think we should probably be consistent here.